### PR TITLE
Adds a version selector to the documentation to match the branches

### PR DESCRIPTION
**Pull request by saltiyazan**

### CODE REVIEW

**Review by ghislainbourgeois — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 site_name: Charmed Magma
-repo_url: https://github.com/canonical/charmed-magma
+repo_url: https://github.com/canonical/charmed-magma/latest
```
Comment (gruyaume) [inline mkdocs.yml:2]: Is this a real URL? I'm aware the PR is already merged

Comment (saltiyazan) [reply mkdocs.yml:2]: Yes the url should work, it did locally 
once you issue `mike deploy` <version>.

Comment (gruyaume) [reply mkdocs.yml:2]: Ok, I get a 404 error when copying this URL in my browser

Comment (saltiyazan) [reply mkdocs.yml:2]: The github action will fix it, soon.
it will run the correct `mike deploy`

**Pull request merged**